src/struct_mv/struct_matrix.c

```diff
@@ -1285,7 +1285,9 @@ hypre_StructMatrixInitializeData( hypre_StructMatrix *matrix,
 {
    if (data == NULL)
    {
-      data = hypre_CTAlloc(HYPRE_Complex, hypre_StructMatrixDataSize(matrix),
+      /* TODO (VPM): we should be able to just do hypre_TAlloc, but that worsens PFMG convergence */
```

Comment:
This may be because of ghost zones.  We need to initialize them to zero and this is the cheapest way to ensure that will be the case.  This might also be something to think about wrt the other CTAlloc() calls changed in this file.

Reply:
After thinking about this more, I think the ghost zones shouldn't matter.  We needed to be more careful about them on the master branch, but I can't think of why we need to worry here.  It would be good to understand why PFMG suffers by making this change.  That may be a symptom of a possibly bigger issue we need to fix.

Reply:
Thanks, Rob! I will investigate this. I realized the issue happens also with a single rank, so it's probably something else other than the ghost zones